core/lib/libtomcrypt/src/hashes/sha2/sha512_256.c

```diff
@@ -0,0 +1,158 @@
+// SPDX-License-Identifier: BSD-2-Clause
+/*
```

Comment:
This copyright block is not in the original file https://github.com/libtom/libtomcrypt/blob/v1.18.0/src/hashes/sha2/sha512_256.c

Please keep only what is upstream, and the SPDX tag.

Reply:
Ok will remove copyright block.